src/main/resources/edu/cornell/kfs/krad/config/CUKRADSpringBeans.xml

```diff
@@ -30,4 +30,46 @@
         </property>
     </bean>
 
+    <bean id="cf.maintainableXMLConversionService" class="edu.cornell.kfs.krad.service.impl.NoOpMaintainableXMLConversionServiceImpl" />
+
+    <bean id="cuMaintainableXMLConversionService" parent="cuMaintainableXMLConversionService-parentBean"/>
+    <bean id="cuMaintainableXMLConversionService-parentBean" abstract="true" class="edu.cornell.kfs.krad.service.impl.CuMaintainableXMLConversionServiceImpl">
+        <property name="conversionRuleFiles">
+            <list>
+                <value>classpath:edu/cornell/kfs/krad/config/MaintainableXMLUpgradeRules.xml</value>
+            </list>
+        </property>
+        <property name="conversionRuleFile" value="classpath:edu/cornell/kfs/krad/config/MaintainableXMLUpgradeRules.xml"/>
```

Comment:
This bean is setting both a "conversionRuleFiles" and a "conversionRuleFile" property, is it necessary for it to set both?